jest-preprocess.js

```diff
@@ -1,4 +1,5 @@
 const babelOptions = {
     presets: ['babel-preset-gatsby'],
+    plugins: ['emotion'],
```

Comment:
A main issue solved was emotion-related as I was seeing an error for `Component selectors can only be used in conjunction with babel-plugin-emotion`.